src/index.js

```diff
@@ -12,4 +13,5 @@ export {
   padLeft,
   randomInteger,
   arrayFill,
+  objectValuesToString
```

Comment:
Please leave a dangling comma here to keep our git history clean :) read [more](http://eslint.org/docs/rules/comma-dangle).